poms/bom/src/main/generated/flattened-reduced-pom.xml

```diff
@@ -8428,6 +8412,78 @@
         <artifactId>quarkus-hazelcast-client-deployment</artifactId>
         <version>4.0.0</version>
       </dependency>
+      <dependency>
+        <groupId>io.quarkiverse.langchain4j</groupId>
+        <artifactId>quarkus-langchain4j-core-runtime-spi</artifactId>
+        <version>0.16.4</version>
+      </dependency>
+      <dependency>
+        <groupId>io.quarkiverse.langchain4j</groupId>
+        <artifactId>quarkus-langchain4j-core</artifactId>
+        <version>0.16.4</version>
+      </dependency>
+      <dependency>
+        <groupId>io.quarkiverse.langchain4j</groupId>
+        <artifactId>quarkus-langchain4j-core-deployment</artifactId>
+        <version>0.16.4</version>
+      </dependency>
+      <dependency>
+        <groupId>dev.langchain4j</groupId>
+        <artifactId>langchain4j</artifactId>
+        <version>0.32.0</version>
+      </dependency>
+      <dependency>
+        <groupId>dev.langchain4j</groupId>
+        <artifactId>langchain4j-core</artifactId>
+        <version>0.32.0</version>
+        <type>test-jar</type>
+        <classifier>tests</classifier>
+      </dependency>
+      <dependency>
```

Comment:
@jamesnetherton On the top of your mind, would any of the following dependencies below generate issues on the platform ? Like aws ? infinispan ?

Reply:
I think the platform generator plugin would force versions that are 'owned' by their respective platform members. E.g Infinispan versions would come from `quarkus-core` & AWS versions from`quarkus-amazon-services`.

Importing `langchain4j-bom` in its entirety is perhaps a bit heavyweight. It's adding some stuff that I'd prefer to not have in the BOM, like lombok, jsoup and a test-jar. Maybe we can tweak the `cq-maven-plugin` config to restrict what's included to `dev.langchain4j:*` (maybe even `*` is too much).

Or, if there's just a known set of dependencies that we think are used commonly, maybe we just add those individually to the BOM.

Reply:
Need more thinking indeed, jsoup & lombok would be too high a price.

Actually, I'm not even sure it would have the desired effect.
Like, let's say this pr is integrated and a next quarkus-platform release embed it.
Would the dependency management include the `quarkus-langchain4j-bom` ? Like  an application depending on quarkus-langchain4j-ollama would have the right version pulled from dependency management ?

Reply:
> Would the dependency management include the `quarkus-langchain4j-bom`

Everything in `camel-quarkus-bom` should in theory make its way into the platform generated `quarkus-camel-bom`. So the langchain4j bits should get propagated.

You can verify it locally by building the platform with the CQ SNAPSHOT.

Reply:
After investigations, `jsoup` and `lombok` are used at deep langchain4j level. Managing them seems unmaintanable. Unless there are other ideas, the best way would be too let users force the versions by importing the `quarkus-langchain4j-bom`.

Reply:
We could maybe add a section to the docs about importing the BOM in user apps. You could add an asciidoc attribute to `antora.yml` for the langchain4j version so that it could be used in a code snippet.

Reply:
That looks like an interesting idea. Let investigate to update this pr.